modules/update-common-terms.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+= Common terms
+
+Control plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines.
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OSUS to view valid updates and update paths based on your current component versions and information in the graph.
```

Comment:
nit: first use of OSUS, so it should get spellled out `the _OpenShift Update Service_ (OSUS)`.  Or the update-service glossary entry below should be shifted up before this entry.

Reply:
also "view" -> "retrieve" or some such?  Because we store the information locally, and "view" sounds less persistent.

Reply:
OSUS is spelled out in the assembly file which comes before this file. So, leaving that as is.